lib/src/map/controller/map_controller_impl.dart

```diff
@@ -418,21 +424,24 @@ class MapControllerImpl extends ValueNotifier<_MapControllerState>
     required Curve curve,
     required bool hasGesture,
     required MapEventSource source,
+    AnimationEndedCallback? onAnimatedEnded,
+    AnimationCancelledCallback? onAnimationCancelled,
```

Comment:
Could there be any use in having these callbacks exposed through `MapOptions`? Again, need to consider if we're adding too many callbacks and whether there's a better way to do it (eg. more event types, remove callbacks except `onMapEvent`) - but that's a discussion for another time.

Reply:
Alowing the user to listen for it makes indeed sense but isn't this already possible with the event system?
Side note: these callbacks are not persistent, they are only for one animation and get cleared afterwards.